programs/marginfi/tests/admin_actions/account_transfer.rs

```diff
@@ -35,6 +44,8 @@ async fn marginfi_account_transfer_happy_path() -> anyhow::Result<()> {
     assert_eq!(account_new.authority, new_authority.pubkey());
     // Old account is recorded as the migration source
     assert_eq!(account_new.migrated_from, marginfi_account.key);
+    // last_update only actualized for the new account - old one stays the same
```

Comment:
again should probably update the old account too, but no strong feelings either way tbh